src/libs/actions/Report.ts

```diff
@@ -543,6 +555,30 @@ function reportActionsExist(reportID: string): boolean {
     return allReportActions?.[reportID] !== undefined;
 }
 
+function updateGroupChatName(reportID: string, reportName: string) {
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: {reportName},
+        },
+    ];
+
+    API.write(WRITE_COMMANDS.UPDATE_GROUP_CHAT_NAME, {reportName, reportID}, {optimisticData});
+}
+
+function updateGroupChatAvatar(reportID: string, file?: File | CustomRNImageManipulatorResult) {
+    // If we have no file that means we are removing the avatar.
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: {avatarUrl: file?.uri ?? ''},
+        },
+    ];
```

Comment:
Created an issue to further improve this [here](https://github.com/Expensify/App/issues/39983).

I can update the `failureData` to just revert the `avatarUrl` back to previous if something goes wrong if anyone feels strongly. But think it can be implemented separately.